### DISCUSSION TITLE
Can the archive page support paging?

### DISCUSSION THREAD

**Discussion by dyxang** [adityatelange/hugo-PaperMod#587] [Q&A] [answered]
If I have too many posts, the archive page will be too long, slow to load, and not very viewable

**Comment by danielfdickinson** [comment, marked as answer]
Hugo only supports pagination on list pages. Archive is a 'single' page so it can't use pagination. It's not a theme issue, it's a limitation of Hugo. See https://discourse.gohugo.io/t/how-to-show-range-with-paginator-partial-on-single-page-templates/31651/2 which references https://gohugo.io/templates/pagination/#list-paginator-pages

**Reply by dyxang** [reply]
Thanks a lot.
